### PR TITLE
chore(docs): Heroku deployment guide

### PR DESCRIPTION
**Pull request by f9n**
Depends on: https://github.com/hertzg/tesseract-server/pull/52

### CODE REVIEW

**Comment by hertzg** [comment]
Hello @f9n Thanks for the PR (again). Don't you think heroku deployment guide would be better as a guide linked from README instead of including this as part of the project?

We could make a section in Readme with `## Deployment Guides` and create a link to `docs/heroku-deploy.md` with the instructions? WDYT? :thinking:

**Comment by f9n** [comment]
Hello. It makes sense. Thanks for your responses. 👍

File: Aptfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# https://stackoverflow.com/questions/66087588/tesseract-error-while-loading-shared-libraries-libarchive-so-13-python
```
Comment (hertzg) [inline Aptfile:1]: Would you mind inlining these files contents into the `heroku-deploy.md`?

File: Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: yarn run build && yarn run run
```
Comment (hertzg) [inline Procfile:1]: Seems like an invalid command :thinking: 

Also 

> Would you mind inlining these files contents into the `heroku-deploy.md`

**Review by hertzg — CHANGES_REQUESTED** [review]

File: Procfile (hunk 2 above)
Comment (f9n) [reply Procfile:1]: I think this is not a problem because my Heroku app works fine.

**Comment by hertzg** [comment]
Thansk for the updates @f9n merged. 🎉

**Pull request merged**